public/javascripts/SVLabel/src/SVLabel/task/TaskContainer.js

```diff
@@ -125,6 +125,11 @@ function TaskContainer (navigationModel, neighborhoodModel, streetViewService, s
         }));
         paths = null;
 
+        // updates the segments that the user has already explored
```

Comment:
Can you change these two comments to conform to the [style guide](https://github.com/ProjectSidewalk/SidewalkWebpage/wiki/Style-Guide#general-guidelines).